scenarios/all_in_one/playbooks/03_openshift_client.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Install OpenShift Client
+  hosts: localhost
+  become: true
+  gather_facts: false
+  tasks:
+    - name: Register with Red Hat Subscription Manager
```

Comment:
Does this also need the pool config as per `3.` in https://docs.openshift.com/container-platform/4.9/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli-rpm_cli-developer-commands

Reply:
With autosuscribe set to true, there is no need to specify the pool.

Reply:
https://docs.ansible.com/ansible/latest/collections/community/general/redhat_subscription_module.html

Reply:
ok cool. perhaps we should use `auto_attach` instead `autosuscribe` as its deprecated.

Reply:
You are right, I have committed new code with auto_attach instead of autosuscribe